sensing/image_diagnostics/README.md

```diff
@@ -0,0 +1,41 @@
+# image_diagnostics
+
+## Purpose
+
+The `image_diagnostics` is a node that check the status of the input raw image.
+
+## Inner-workings / Algorithms
+
+## Inputs / Outputs
+
+### Input
+
+| Name              | Type                      | Description |
+| ----------------- | ------------------------- | ----------- |
+| `input/raw_image` | `sensor_msgs::msg::Image` | raw image   |
+
+### Output
+
+| Name                                | Type                      | Description                           |
+| ----------------------------------- | ------------------------- | ------------------------------------- |
+| `image_diag/debug/gray_image`       | `sensor_msgs::msg::Image` | gray image                            |
+| `image_diag/debug/dft_image`        | `sensor_msgs::msg::Image` | discrete Fourier transformation image |
+| `image_diag/debug/diag_block_image` | `sensor_msgs::msg::Image` | each region state colorization        |
+
```

Comment:
[MUST] I think this node output "/diagnostic" topic. Also, "image_diag/image_state_diag" and "frequency_intensity_ratio", They should be written here.

Reply:
Could you add "/diagnostic" topic to output ? Do you have concerns about adding it?

Reply:
Sorry, I missed it. Already added.